src/doc-printer.js

```diff
@@ -35,7 +62,11 @@ function fits(next, restCommands, width) {
 
           break;
         case "indent":
-          cmds.push([ind + doc.n, mode, doc.contents]);
+          cmds.push([indent(ind), mode, doc.contents]);
```

Comment:
It's a little confusing to name these functions `indent` and `align`, the same as the "commands". Can you possibly rename these? At first I thought it was calling the command and was very confused. Maybe something like `makeIndent`?